frontend/.gitignore

```diff
@@ -22,3 +22,6 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+DS_Store
+.env
```

Comment:
We won't need a .env file on the frontend as the variables are all related to backend tasks (api, database, etc) and it's not as secure to keep these in the frontend of a project as they can be accessed easier. 

Go ahead and remove the .env file in the frontend folder, as well as remove the .env from the .gitignore file here as it won't be needed.